### ISSUE TITLE
D7.5.1 Technical-oriented talk about the principles and benefits of the ModelWriter-ITEA approach and tooling

### ISSUE THREAD

**Issue by ferhaterata** [ModelWriter/Deliverables#117]
This is an issue of the task : T7.5 Community Forum & Open Source Campaign (OBEO + UNIT)
-----------------------------------------------------------------------------------------------------------------------------

The ModelWriter-ITEA project will not only produce new and important tools and techniques, but will introduce the concept of Text & Model synchronisation. We believe this will provide a step change in the way of software engineering and consequently systems will be developed and so expect far-reaching impact well beyond the immediate project goals. In particular, we expect this to become vital in future systems, both within space applications and elsewhere.
In order to foster this view and to develop this new field, we will engage a campaign to develop a vibrant user community for all the Open Source technologies produced in the context of ModelWriter. This campaign will address and explain questions such as: What is reconfigurable text & model synchronisation? Why is it useful? How can we use it? How has it been used already? And so on.
Work on this user community will culminate in a set of talks given at open-source technical conferences (such as the EclipseCon or the Agile ALM connect conferences - D7.5.1) and scientific conferences (such as Models - D7.5.2) during the last year of the project. The aim of these talks will be to create an independent, self-sustaining and long-lived user community around the ModelWriter-ITEA project, and to evangelize the principles and benefits of the approach to both industrial and research worlds.

**Comment by ejuliot** [comment]
The deliverable is here: https://github.com/ModelWriter/Deliverables/tree/master/WP7
The initial version of the technical presentation is here: https://github.com/ModelWriter/WP7/tree/master/Documents
It would nice it we can use the same fragment to consolidate our technical presentation of MW technologies. So, this document will be updated continuously, each time we need to do a talk.

**Comment by ylussaud** [comment]
I just had a look at the presentation it's great. I just noticed few problems:
- slides numbers are not present/consistent
- the slide numbered 25/25 doesn't make sens. It should be removed or replace by a sketch of the view showing links.
- is the consortium slide up to date ?

**Comment by ferhaterata** [comment]
I liked the presentation. Can we use a format like the one of which Etienne presented in Atsen 2014, Istanbul? A format similar to that one is quite good for the graphics that spans the entire page and the text in front.  

BTW, Page number 12 that shows the partners on the Europa map is old as Yvan indicated. Let me talk with the ITEA office and request them to generate up-to-date one.

**Comment by ejuliot** [comment]
Thanks for your feedback.
I updated the document with these elements:
* I removed the number of slides. That is a bad practices, specially when you have plenty of pictures because some attendees can be afraid of being boring by too many slides.
* for slides 6 and 14, I did a copy of a printscreen because I didn't have the orginal. Ferhat, if you have them, please replace them to be able to edit them later.
* slide 25/25: I remove it. Please add the sketch you spoke about.

For the consortium slide, update it please as soon as the ITEA regenerate it.

**Comment by eraytuzun** [comment]
Hi Etienne,

It look great overall, just minor comments

- Agree with other comments, consortium slide(Slide 12) needs to be
updated. It does not have HAVELSAN in it:)
- Slide 3 usefull -> useful
- Slide 4 "hard to keep it up-to-dated" -> "hard to keep it up-to-date"
- Slide 8 "texts connector" ->"text connector"

Eray


Eray Tüzün, PhD
Principal Software Engineer
Havelsan
http://eraytuzun.com

**Comment by ejuliot** [comment]
Thanks Eray.
Updated.

Etienne JULIOT
Vice President, Obeo

**Comment by ferhaterata** [comment]
Hi @ejuliot, @ylussaud and @eraytuzun,

I've recently updated the slides in order to present it on a local event in Izmir. I have used all the graphics that @ejuliot added in the presentation. In addition to that, I mainly focused on the model and text editor usage scenarios in System/Software Engineering contexts, which I believe that helps audience understand what is the ultimate goal of the project in terms of model & text Synchronization and Semantic Parsing of a text expressed in natural language.   

https://github.com/ModelWriter/WP7/blob/master/Documents/ModelWriter%20Presentation%20v2.0.pdf

Thanks.
